src/AppInstallerCommonCore/Public/winget/ExperimentalFeature.h

```diff
@@ -23,7 +23,8 @@ namespace AppInstaller::Settings
             Dependencies = 0x1,
             // Before making DirectMSI non-experimental, it should be part of manifest validation.
             DirectMSI = 0x2,
-            Max, // This MUST always be after all experimental features
+            PortableInstall = 0x4,
+            Max = 0x8, // This MUST always be after all experimental features
```

Comment:
Setting a value for `Max` makes mistakes more likely in the future.  If you want to make the fact that these are flags clearer, put a comment on the enum.

Reply:
I'd also add that it might be clearer that it's a bit flag if the bit shift syntax were used instead of the hex representation -
```
            None = 0,
            Dependencies = 1 << 0,
            // Before making DirectMSI non-experimental, it should be part of manifest validation.
            DirectMSI = 1 << 1,
            PortableInstall = 1 << 2,
            Max , // This MUST always be after all experimental features

Reply:
Removed value from `Max`